lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -139,7 +170,7 @@
       - The order of the list is significant in determining the value assigned to a
         variable that shows up more than once.
       - If variable also present in I(env), then I(env) value will override.
-    version_added: "2.8"
+    version_added: 2.8
```

Comment:
You shouldn't change `version_added` for existing options, validate-modules doesn't like that.

Reply:
Ah that explains the sanity check errors. Tried to make all `version_added` quoted and it returned errors. Reverting.